packages/nextjs/src/config/templates/dataFetchersLoaderTemplate.ts

```diff
@@ -0,0 +1,28 @@
+import type {
+  GetServerSideProps as GetServerSidePropsFunction,
+  GetStaticPaths as GetStaticPathsFunction,
+  GetStaticProps as GetStaticPropsFunction,
+} from 'next';
+
+import type { withSentryGSPaths, withSentryGSProps, withSentryGSSP } from '../wrappers';
+
+// TODO: Somehow the types for the imported SDK aren't recognized when it's imported using `require`, so for the moment,
+// we're faking it
+type SentrySDK = {
+  withSentryGSSP: typeof withSentryGSSP;
+  withSentryGSProps: typeof withSentryGSProps;
+  withSentryGSPaths: typeof withSentryGSPaths;
+};
+
+declare const _getServerSideProps: GetServerSidePropsFunction;
+declare const _getStaticProps: GetStaticPropsFunction;
+declare const _getStaticPaths: GetStaticPathsFunction;
+
+// TODO: This is a require rather than an imoprt because for some reason, using `import` makes rollup silently refuse to
+// output a built version of this file.
+// eslint-disable-next-line @typescript-eslint/no-var-requires, import/no-extraneous-dependencies
+const ServerSideSentryNextjsSDK = require('@sentry/nextjs');
```

Comment:
Nope, not yet. That's why I left it as a `TODO`. I did try marking `@sentry/nextjs` external, and that had its own problems, IIRC, though I don't remember exactly what. It felt lower priority, though, since it's not a blocker, so in the interests of time I left it. I'll throw another 10 min at the problem and at minimum see if I can come up with better context for the comment.